dp/subsetsum.py

```diff
@@ -0,0 +1,20 @@
+def isSubsetSum (items, length, summ):
```

Comment:
Please provide the comment stating what does the module/function do. Also, include the doctest in the comments.
Doctest reference - https://docs.python.org/2/library/doctest.html